index.html

```diff
@@ -37,24 +45,65 @@ <h2>I'm a Software Developer</h2>
 
         <article class="extra">
           <section class="works-centered">
-            <h2 class="works-title">My Recent Works</h2>
-            <hr>
-            <img src="./logos/My Recent Works.png" alt="My Recent Works">
-            <h2 class="multi-stories">Multi-Post Stories</h2>
-            <p>
-              A daily selection of privately personalized reads; no accounts or sign-ups 
-              required. has been the industry's standard dummy text ever since the 1500s, 
-              when an unknown printer took a standard dummy text.
-            </p>
+            <div class="top-works">
+              <h2 class="works-title">My Recent Works</h2>
+              <hr>
+            </div>
+
+            <div class="main-works">
+              <img src="./logos/My Recent Works.png" alt="My Recent Works">
+              <div class="main-works-1">
+                <h2 class="multi-stories">Multi-Post Stories</h2>
+                <p>
+                  A daily selection of privately personalized reads; no accounts or sign-ups 
+                  required. has been the industry's standard dummy text ever since the 1500s, 
+                  when an unknown printer took a standard dummy text.
+                </p>
+              </div>
+            </div>
+
             <ul class="project-list">
               <li><a class="start-list" href="#">HTML</a></li>
               <li><a class="second-list" href="#">CSS</a></li>
               <li><a class="third-list" href="#">Javascript</a></li>
             </ul>
             <button>See Project</button>
           </article>
+        
+        <div class="print-container">
+          <article class="print-data data-0">
+            <button>See Project</button>
+          </article>
```

Comment:
- [ ] Please try as much as possible to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=E5AVHIf7a3eVNCi2-0). There should be a spacing on the left of the cards to ensure that your portifolio is more user-friendly ☺️.
- [ ] In addition, the spacing between the cards should be reduced to match the Figma design ☺️.
Your design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/b75f73a6-3d0f-4141-b222-669e799165cd)

Figma design 😎 
![image](https://github.com/anyars-encarta/Porfolio/assets/99394374/e593cb2d-df3c-4f5b-bd0c-3de5e414e6b1)